gaelib/utils/web.py

```diff
@@ -63,58 +79,53 @@ def log_request_info():
 def inject_global_template_vars():
   return dict(app_name=get_app_or_default_prop('APP_NAME'),
               auth_config=get_auth_config(),
-              dashboard_notification_admin=get_app_or_default_prop('DASHBOARD_NOTIFICATION_ADMIN'),
-              dashboard_prefix=get_dashboard_url_prefix(), env=get_env(),
-              sidebar_template=get_sidebar_template(),
-              )
+              dashboard_notification_admin=get_app_or_default_prop(
+      'DASHBOARD_NOTIFICATION_ADMIN'),
+      dashboard_prefix=get_dashboard_url_prefix(), env=get_env(),
+      sidebar_template=get_sidebar_template(),
+  )
 
 
-def startup(auth=True, parameter_logging=False, client_logging=False, dashboard=True, verification=True):
+def startup(auth=True, blueprint_scope='', parameter_logging=False, client_logging=False, dashboard=True, verification=True):
   """The startup script to create flask app and check if
       the application is running on local server or production.
   """
 
   if is_dev():
-    # os.environ['DATASTORE_EMULATOR_HOST'] = '172.17.0.2:8888'
-    # if get_app_or_default_prop('DATASTORE_PROJECT_ID'):
-    #   os.environ['DATASTORE_PROJECT_ID'] = get_app_or_default_prop('DATASTORE_PROJECT_ID')
-    # if get_app_or_default_prop('GOOGLE_CLOUD_PROJECT'):
-    #   os.environ['GOOGLE_CLOUD_PROJECT'] = get_app_or_default_prop('GOOGLE_CLOUD_PROJECT')
-
-    # Dumb hack we need to run locally
+    # Dumb hack we need to run the app locally
     # The private key that's in this file is not associated with
     #  a real account
     os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = os.getcwd() + \
```

Comment:
why don't we add the environment var for DATASTORE_EMULATOR_HOST, DATASTORE_PROJECT_ID AND GOOGLE_CLOUD PROJECT? Those are necessary for running the app in the emulator


    os.environ['DATASTORE_EMULATOR_HOST'] = 'localhost:8089'
    os.environ['DATASTORE_PROJECT_ID'] = <any project id>
    os.environ['GOOGLE_CLOUD_PROJECT'] = <same project id as above>

Reply:
We've set it up as part of the docker run command in run_test_container.sh in gaelib or run_dev_container.sh in the app repositories.